uw-frame-components/portal/main/partials/username.html

```diff
@@ -15,14 +15,23 @@
     <span>{{sessionCtrl.firstLetter}}</span>
     <md-tooltip class="top-bar-tooltip" md-direction="bottom" md-delay="500">Options</md-tooltip>
   </md-button>
-  <md-menu-content width="3">
-    <md-menu-item ng-if="$storage.showSettings">
+  <md-menu-content width="5" class="top-bar-menu-content username-menu-content">
+    <md-menu-item class="md-2-line">
+      <div class="md-list-item-text">
+        <p class="display-name">{{ sessionCtrl.username }}</p>
+        <p class="campus-id" ng-if="sessionCtrl.campusId != ''">Campus ID: {{ sessionCtrl.campusId }}</p>
```

Comment:
Since [`''` is falsey](https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript), would this be more idiomatic as `... ng-if="sessionCtrl.campusId">`? And safer, since all the other falsey values would do the right thing too?